service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/ServiceRegistryTaskInitializer.java

```diff
@@ -1,4 +1,4 @@
-/*
+package org.apache.servicecomb.serviceregistry.registry;/*
```

Comment:
The package should add below the License header.

Reply:
oh, this happened randomly
do you know why......

Reply:
done